pkg/application/interfaces.go

```diff
@@ -57,3 +57,7 @@ type blueprintSpecRepository interface {
 type remoteDoguRegistry interface {
 	domainservice.RemoteDoguRegistry
 }
+
+type maintenanceMode interface {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>type `maintenanceMode` is unused

Reply:
This is only because we need to redeclare a interface to generate mocks. This interface is not really needed for the production code.